lib/tropic/tropic_paragraph.flow

```diff
@@ -912,6 +923,45 @@ reflowAndFitTParaWords(
 	}
 }
 
+detachWord(word : TParaWord, detach : bool) -> void {
+	nextDistinct(word.inspector.detached, detach);
+}
+
+rtrimWords(words : [TParaWord]) -> [TParaWord] {
```

Comment:
this one seems should depend on rtl? I mean, in RTL it might be wrong to trim right space. see paragraph3 for example

Reply:
It not depends on rtl in tropic_paragraph. It works correctly. I tested it